mk/utils/glide.mk

```diff
@@ -0,0 +1,28 @@
+# Cross builder helper
+define glide-install
+	glide install -s --update-vendored
+endef
+
+define glide-update
+	glide update -s --update-vendored
```

Comment:
Forgot to mention.   We don't need the sources history commited along with the git repo... In fact, I think git make an entry for the folder as if it's a sub git repo... which doesn't help us on fresh machines where we are cloning the repo from scratch.